### DISCUSSION TITLE
Issue using iPad with connected display.

### DISCUSSION THREAD

**Discussion by elminster3k** [blinksh/blink#1740] [Support] [answered]
I would very much like to connect a display, and have blink running on the iPad while I watch a movie on the connected display.  The issue for me is when I have Blink open on the ipad, the second display immediately gets another full screen blink shell that I can't do anything with.  If I move blink to the second display it behaves normally, but any time I have it open or up on the iPad it seems to take over the other screen.  Not sure how to disable this?

**Comment by carloscabanero** [comment, marked as answer]
Blink supports multiple modes, and by default we still support the old "extended display". To make the second display behave "normal" with Stage Manager, go to config > Appearance, and choose "Mirror" as the default setting. Restart Blink and it should be good.
